src/main/kotlin/org/jitsi/nlj/transform/node/SrtpTransformerNode.kt

```diff
@@ -98,6 +98,7 @@ class SrtpTransformerNode(name: String) : MultipleOutputTransformerNode(name) {
         }
     }
 
+    private var num_srtp_processed = 0
```

Comment:
I noticed the other day warnings about the style of these variable names btw...why are the with underscores?

Reply:
To match the json - I can change them.